msu/squirrel_hooks/mod_hooks.nut

```diff
@@ -154,6 +154,28 @@ local _mods_runQueue = ::_mods_runQueue;
 			{
 				if (dependencyMod == null)
 				{
+					switch (dependencyTable.Name)
```

Comment:
A table with `dlcCode` : `dlcName` and an `in` check would probably look cleaner.

Reply:
This is just a patchy solution for the time being so I've kept it rather simple especially because all the keys and values are constants and don't need changing. For the future we should consider making a system where mod ids can be matched with mod names maybe via an online file or something.